### PR TITLE
:sparkles: Adjusts the textview size. allows user to copy

### PR DESCRIPTION
**Pull request by ammon-lockwood**
Adjust the textview size and allows the user to copy the encoded and decoded messages. #12

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# mac os x
+.DS_Store
```
Comment (ammon-lockwood) [inline .gitignore:107]: We don't want to have these files in our repo so I added it to be ignored.

File: app.py [hunk 2]
```diff
@@ -18,13 +18,18 @@ def __init__(self):
...
         self.set_titlebar(header)
 
+        # TODO: layout would paned or stack work better than flowbox?
         flowbox = Gtk.FlowBox()
         flowbox.set_valign(Gtk.Align.START)
         flowbox.set_max_children_per_line(30)
         flowbox.set_selection_mode(Gtk.SelectionMode.NONE)
 
+        # TODO: add input for the shift
+        # Gtk.Scale or Gtk.SpinButton?
```
Comment (ammon-lockwood) [inline app.py:28]: coming in the PR #16

File: app.py [hunk 3]
```diff
@@ -18,13 +18,18 @@ def __init__(self):
...
+
         # start decoder
         self.message_to_decode = Gtk.TextView()
+        self.message_to_decode.set_size_request(300,50)
```
Comment (ammon-lockwood) [inline app.py:32]: This sets the size of the text box so it's large enough to make it easier to work with.

File: app.py [hunk 4]
```diff
@@ -36,13 +41,16 @@ def __init__(self):
         decode_button.connect("clicked", self.on_decode_clicked)
 
         self.decoded_label = Gtk.Label()
+        self.decoded_label.set_selectable(True)
```
Comment (ammon-lockwood) [inline app.py:44]: This allows us to select the text so we can copy it and paste if we wanted. Most important for the encode label

**Review by aprilchatland — APPROVED** [review]

**Pull request merged**